src/types/index.ts

```diff
@@ -120,3 +120,5 @@ export enum Sync {
   Syncing,
   Synced,
 }
+
+export type BondFor = 'Pool' | 'Nominator';
```

Comment:
Is there a reason these strings are capitalised? Maybe just have all lower case, one less complexity.

Reply:
like export type bondfor = 'pool' | 'nominator'; ?

Reply:
just the string literal types, not the type name.